src/components/admin/AdminDev.js

```diff
@@ -67,16 +77,58 @@ export default class AdminDev extends React.Component {
       key: tabEventKeys.config,
       title: 'Config'
     }, {
+      key: tabEventKeys.user,
+      title: 'User'
+    }, {
       key: tabEventKeys.readOnlyMatches,
       title: 'ReadOnlyMatches'
+    }, {
+      key: tabEventKeys.players,
+      title: 'Spelers'
+    }, {
+      key: tabEventKeys.admin,
+      title: 'Admin'
     }];
 
     return (
-      <TabbedContainer
-        style={{marginTop: 10, marginBottom: 20}}
-        openTabKey={tabEventKeys.matches}
-        tabKeys={tabConfig}
-        tabRenderer={this._renderSection} />
+      <div style={{padding: 5}}>
+        <TabbedContainer
+          style={{marginTop: 10, marginBottom: 20}}
+          openTabKey={tabEventKeys.matches}
+          tabKeys={tabConfig}
+          tabRenderer={::this._renderSection} />
```

Comment:
dit was je probleem: die "::this._renderSection" de "::" is een shortcut voor `this._renderSection.bind(this)`

door die bind(this) zorg je ervoor dat de huidige context beschikbaar blijft binnenin de _renderSection